packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -141,35 +149,70 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 	return (
 		<Stack className="container-summary-view">
 			<StackItem>
-				<span>
-					<b>Container</b>:{" "}
-					{containerState.nickname !== undefined
-						? `${containerState.nickname} (${containerState.id})`
-						: containerState.id}
-				</span>
-			</StackItem>
-			<StackItem>
-				<span>
-					<b>Status</b>: {statusString}
-				</span>
-			</StackItem>
-			<StackItem>
-				{containerState.clientId === undefined ? (
-					<></>
-				) : (
-					<span>
-						<b>Client ID</b>: {containerState.clientId}
-					</span>
-				)}
-			</StackItem>
-			<StackItem>
-				{containerState.audienceId === undefined ? (
-					<></>
-				) : (
-					<span>
-						<b>Audience ID</b>: {containerState.audienceId}
-					</span>
-				)}
+				<Table>
+					<TableBody>
+						<TableRow>
+							<TableCell>
+								<TableCellLayout>
```

Comment:
I... wow 😆 . Wondering if the FluentUI react components already has a way to make this less verbose or if we might want to code a small helper of ours at some point. Something like "Need a plain table with no special styles? Just give me the contents of the cells as an array of arrays of React fragments".